Configuration/StandardSequences/python/GeometryConf.py

```diff
@@ -32,6 +32,7 @@
     'TrackerOnly' : 'Configuration.Geometry.GeometrySimTracker_cff,Configuration.Geometry.GeometryRecoTracker_cff',
     'HCal'        : 'Configuration.Geometry.GeometrySimHCAL_cff,Configuration.Geometry.GeometryRecoHCAL_cff',
     'Extended2023sim'   : 'Extended2023sim,Extended2023simReco',
-    'Extended2023dev'   : 'Extended2023dev,Extended2023devReco'
+    'Extended2023dev'   : 'Extended2023dev,Extended2023devReco',
+    'Extended2023LReco'   : 'Extended2023LReco,Extended2023LRecoReco'
```

Comment:
@boudoul - IMHO, it would be better to introduce new configurations *when* they are different. If the logic is that 2023sim configuration is for (all sub detectors that can run) SIM step and 2023LReco is for (all sub detectors that can run) Local reconstruction, LReco workflow as defined needs exactly the same sim geometry scenario as it runs L1 digi simulation. If there is any difference, it would be in producer configurations, not geometry's.